widgets/plugins/src/PluginsTable.tsx

```diff
@@ -142,7 +142,7 @@ const PluginsTable: FunctionComponent<PluginsTableProps> = ({ data, toolbox, wid
                             key={item.id}
                             selected={item.isSelected}
                             onClick={() => selectPlugin(item)}
-                            onMouseOver={setHoveredPlugin}
+                            onMouseOver={setHoveredPlugin as any} // TODO: How does it work? :(, probably a bug
```

Comment:
the whole "hoveredPlugin" seems to be a noop

Reply:
Seeing how `hoveredPlugin` is used (L149), shouldn't we have here:
```suggestion
                            onMouseOver={_event => setHoveredPlugin(item.id)}
```
? 🤔

Reply:
I'd avoid functional changes that don't seem to fix any bugs in a refactor.

Could you verify that whole useState of hoveredPlugin is an elaborate noop, so we can just delete it?

Reply:
I don't have enough time to get too deep, but from what I saw it seems like it needs a fix and I'm almost sure that the fix I provided would do the trick (but as always - I might be wrong 🙂).

As I really want to close this PR soon please do the following:
1. Create a ticket in Jira (type: Task, label: UI, stage) to investigate and provide the solution for `hovered*` states (the problem is not only here in this file, but also in other files - [widgets/deployments/src/DeploymentsSegment.tsx](https://github.com/cloudify-cosmo/cloudify-stage/blob/38c0689708f4254b3ce294f5a21c0e09bbc52f4d/widgets/deployments/src/DeploymentsSegment.tsx), [widgets/deployments/src/DeploymentsTable.tsx](https://github.com/cloudify-cosmo/cloudify-stage/blob/38c0689708f4254b3ce294f5a21c0e09bbc52f4d/widgets/deployments/src/DeploymentsTable.tsx)
2. Send me new ticket ID, so I'll try to plan it somehow
3. Add `@ts-expect-error` or `@ts-ignore` comment in all 3 places with e.g. the following description: `TODO(RD-xxxx) Set hovered state properly` or something similar. The most important is to provide related ticket ID in the brackets, so anyone can get more details about the problem by reading Jira ticket.

Reply:
https://cloudifysource.atlassian.net/browse/RD-6366

sorry if i am too stingy about this, I'm just very strict about not changing functinality in refactor.

Reply:
Thanks for creating the ticket. It would be better if you could add more description there (if you or someone else look at this ticket in the future he/she might not understand what was the idea). Also you mentioned there `WidgetTable.tsx`, but there's no such file in `cloudify-stage`, so please fix that. In p. 3 I asked to add TODO in all 3 places. I see only one file changed, but... OK, as this PR is taking too long let it be like that.